commands.yaml

```diff
@@ -1,83 +1,70 @@
 help:
 exit:
 quit:
-aad login:
-aad connect:
-aad logout:
-aad disconnect:
+login:
+logout:
 aad oauth2grant add:
 aad oauth2grant list:
 aad oauth2grant remove:
 aad oauth2grant set:
 aad sp get:
-aad status:
-azmgmt flow environment get:
-azmgmt flow environment list:
-azmgmt flow export:
-azmgmt flow get:
-azmgmt flow list:
-azmgmt flow run get:
-azmgmt flow run list:
-azmgmt login:
-azmgmt connect:
-azmgmt logout:
-azmgmt disconnect:
-azmgmt status:
-graph groupsetting add:
-graph groupsetting get: 
-graph groupsetting list:
-graph groupsetting remove:
-graph groupsetting set:
-graph groupsettingtemplate get:
-graph groupsettingtemplate list:
-graph login:
-graph connect:
-graph logout:
-graph disconnect:
-graph o365group add:
-graph o365group get:
-graph o365group list:
-graph o365group remove:
-graph o365group restore:
-graph o365group set:
-graph o365group user add:
-graph teams user add:
+flow environment get:
+flow environment list:
+flow export:
+flow get:
+flow list:
+flow run get:
+flow run list:
+o365groups groupsetting add:
+o365groups groupsetting get: 
+o365groups groupsetting list:
+o365groups groupsetting remove:
+o365groups groupsetting set:
+o365groups groupsettingtemplate get:
+o365groups groupsettingtemplate list:
+o365groups add:
+o365groups get:
+o365groups list:
+o365groups remove:
+o365groups restore:
+o365groups set:
+o365groups user add:
+teams user add:
 graph schemaextension add:
```

Comment:
@waldekmastykarz, that should remain `graph` and will be very alone in its own folder for now :)